etna/whatson/models.py

```diff
@@ -685,6 +685,34 @@ def primary_access_type(self):
         if primary_access := self.event_access_types.first():
             return primary_access.access_type
 
+    @cached_property
+    def date_time_range(self):
+        delta = self.end_date - self.start_date
+        # One session on one date where start and end times are the same
+        # return eg. Monday 1 January 2024, 19:00
+        if (self.start_date == self.end_date) and (len(self.sessions.all()) == 1):
```

Comment:
Question: `start_date` and `end_date` are `DateTimes`, not `Dates` - it's possible that this comparison could fail if the times are different. What effect will this have?

Reply:
If the times are different, we want this comparison to fail - instead we move to the logic of one date with different start and end times.